README.md

```diff
@@ -17,17 +17,14 @@ Diablo 2 and its content is ©2000 Blizzard Entertainment, Inc. All rights reser
 
 ALL OTHER TRADEMARKS ARE THE PROPERTY OF THEIR RESPECTIVE OWNERS.
 
-## Building
+## Building from source
 
-To pull the project down, run `go get github.com/OpenDiablo2/OpenDiablo2`
+Assuming you have Diablo II installed in the default directoy and a working [Go](https://golang.org/doc/install) environment
 
-On windows this folder will most likely be in `C:\users\(you)\go\src\github.com\essial\OpenDiablo2`
-
-In the root folder, run `go get -d` to pull down all dependencies.
-
-To run the project, run `go run .` from the root folder.
-
-You can also open the root folder in VSCode. Make sure you have the `ms-vscode.go` plugin installed.
+```
+go install github.com/OpenDiablo2/OpenDiablo2
```

Comment:
Either use `go get` or add a `git clone` command above to retrieve the source code repository.

Reply:
`go install` actually puts the code into your `$GOPATH` the same way a clone would.

Reply:
Or am I mistaken?

Reply:
It does, I was mainly referring to the clone of the repo. So if you want to get started with OpenDiablo2, then you don't have the repository cloned yet to your local machine. `go get` can clone the repo, or we can clone it; e.g.

```
git clone https://github.com/OpenDiablo2/OpenDiablo2
cd OpenDiablo2
go install
```

That would work

Or we can run
```
go get github.com/OpenDiablo2/OpenDiablo2
```

Thinking about it, the first one is probably preferred, as we use Go modules.

Reply:
Because of modules now, I actually prefer the `git clone` as I want to put the project somewhere I'm going to `work` on it.  If you use `go get`, it will put it in your packages and not likely the working copy you intend to edit.  We should also be encouraging people to do the fork as well, and then update instructions for them to clone their fork instead.  Thoughts?

Reply:
Yeah, lets go with Git clone.

I don't think we need anything specific for fork instructions.

Just add something like this:

```
git clone https://github.com/OpenDiablo2/OpenDiablo2
git remote add myusername git@github.com:myusername/OpenDiablo2.git
```

Then, build like normal:

```
cd OpenDiablo2
go install
```

And when you make changes, pull to your own fork.

```
git checkout -b feature-foo
git commit -a -m 'add feature foo'
git push myusername feature-foo
```

And then submit a PR as normal.